code/style.css

```diff
@@ -2,6 +2,247 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  height: 100%;
+  margin: 0;
+  font-family: Karla, sans-serif;
+  background-image: url(build-set.jpeg);
+}
+
+header {
+  width: 100%;
+  height: 60vh;
+  position: relative;
+}
+
+/* ---------Hero video styling---------- */
+
+#header-video {
+  width: 100%;
+  height: 60vh;
+  object-fit: cover;
+}
+
+.header-content {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+/* ----------Site content styling---------- */
+
+.text-section {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(255, 235, 239);
+  margin: 0;
+  border-bottom: goldenrod 3px solid;
+}
+
+.introduction {
+  padding: 10px 10px 0;
+}
+
+.content-container {
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+  justify-content: center;
+  align-items: inherit;
+}
+
+h3 {
+  text-transform: uppercase;
+}
+
+/* ----------Form styling---------- */
+
+.signup-form {
+  margin-bottom: 20px;
+  padding: 20px;
+  background-color: rgb(252, 203, 210);
+  -webkit-border-radius: 15px;
+  -moz-border-radius: 15px;
+  border-radius: 15px;
+}
+
+.input-container {
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+}
+
+fieldset {
+  padding: 10px;
+  border: purple 2px solid;
+}
+
+legend {
+  font-size: 18px;
+  text-align: center;
+}
+
+#fname,
+#lname,
+#email {
+  padding: 8px;
+  -webkit-border-radius: 15px;
+  -moz-border-radius: 15px;
+  border-radius: 15px;
+}
+
+#fname:focus,
+#lname:focus,
+#email:focus {
+  padding: 8px;
+  -webkit-border-radius: 15px;
+  -moz-border-radius: 15px;
+  border-radius: 15px;
+  background-color: rgb(250, 224, 228);
+}
+
+/* ----------Accordion styling---------- */
+
+.accordionParent {
+  margin-top: 10px;
+  padding: 10px;
+  justify-content: center;
+  background-color: rgb(252, 203, 210);
+  -webkit-border-radius: 15px;
+  -moz-border-radius: 15px;
+  border-radius: 15px;
+}
+
+.accordion-header {
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+}
+
+.question {
+  padding: 10px;
+  text-align: left;
+  transition: 0.4s;
+  text-transform: uppercase;
+  -webkit-border-radius: 15px 15px 0px 0px;
+  -moz-border-radius: 15px 15px 0px 0px;
+  border-radius: 15px 15px 0px 0px;
+}
+
+.question:nth-of-type(8) {
```

Comment:
could these tags be explained perhaps so its even easier for those who reads your code to understand why your website looks so good?

Reply:
Thanks Elsa:) Yes, absolutely, you're so right! I'll add some extra comments for the code to be easier to understand and read.